src/Jackett/Indexers/BakaBT.cs

```diff
@@ -95,12 +95,17 @@ public BakaBT(IIndexerManagerService i, IWebClient wc, Logger l, IProtectionServ
             {
                 CQ dom = response.Content;
                 var rows = dom[".torrents tr.torrent"];
```

Comment:
If you change it to 
`var rows = dom[".torrents tr.torrent, .torrents tr.torrent_alt"];`
it should have the same effect